pandas/core/indexing.py

```diff
@@ -1461,6 +1461,9 @@ def _getitem_axis(self, key, axis=0):
             if isinstance(labels, MultiIndex):
                 if (not isinstance(key, tuple) and len(key) > 1 and
                         not isinstance(key[0], tuple)):
+                    if isinstance(key, ABCSeries):
+                        # GH 14730
```

Comment:
I thought taking the `.values` was more explicit, but `list` is probably more familiar. Changed.